config-schema.json

```diff
@@ -455,6 +455,21 @@
                 ],
                 "pattern": ""
               },
+              "custom_features": {
+                "$id": "#/properties/entities/properties/remote/items/properties/custom_features",
+                "type": "array",
+                "title": "The custom_features Schema",
+                "items": {
+                  "$id": "#/properties/entities/properties/remote/items/properties/custom_features/items",
+                  "type": "string",
+                  "title": "The Custom Features Schema",
```

Comment:
Please add a descriptive title that matches the item.
This can improve validation error messages and allows us to create auto-generated edit forms in the future.

Reply:
OK, i just imitated the other titles :)